### ISSUE TITLE
Error replay TF1 (Runtime Error) 

### ISSUE THREAD

**Issue by FrozenWarwick** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#191]
Hi there,

Just noticed that TF1 replay is not working when selecting the menu TF1 it popup an error message (Runtime error) Unable to find requested section with tag of 'ul' and attributes of class...

I have: 
Fully re-install catchup, no changes
Fully re-install KODI no changes

Start all from scratch, no changes.
I read the log and probably a syntax issue but not sure.

Let me know your througt.
Thanks 

[Runtime_error-TF1-REPLAY.txt](https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/files/3292681/Runtime_error-TF1-REPLAY.txt)

**Comment by FrozenWarwick** [comment]
Forgot to say but you can see it in the log file, I'm running on rasp 3b+.
Cheers

**Comment by wwark** [comment]
Hi @FrozenWarwick,

Fixed in the last released https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/releases/tag/v0.2.8

Thanks

**Comment by FrozenWarwick** [comment]
Hi @wwark,

Great thanks a lot. How do I apply the fix? Do I have to download the file manualy and past on the current folder? Use Git or the update will appear into the catchuptv plugin interface?

Let me know.
Thanks.

**Comment by FrozenWarwick** [comment]
Hi @wwark,

I have replaced the current folder by the fixed one then rebooted the raspberry and it works properly?
Is it the correct method to do so?

Let me know.
For the time being thanks a lot for your support.
Thanks.

**Comment by wwark** [comment]
hi @FrozenWarwick,
Normally, it will update if you install our repository.
If it is not the case I propose you to follow this documentation https://catch-up-tv-and-more.github.io/installation/

The last release 0.2.8 contains fixe for MyTF1.

Thanks,
Wwark

**Comment by FrozenWarwick** [comment]
Hi @wwark,
Ok noted, I have followed exactly this tuto to install it but when I checked from the plugin the section "Update" the 0.2.8 version didn't popup so I ended replacing the current folder by the new one and it applied your fix. TF1 works perfectly now.

Thanks again for your fast support, really appreciate.

A little things can be improved for the navigation's menu, on the first menu you have several option like "Live TV" or "Replay". If you get into "Replay" then go deep until the movie you want to play, one more last "Replay" menu is displayed again but you are already in Replay... For the navigation this last one is not necessary but again if the second replay menu is there it is probably for a good reason.

Let me know if what I say is stupid or not.
 
Thanks,
FW
